credentials-operator/templates/credentials-operator-mutating-webhook-configuration.yaml

```diff
@@ -22,7 +22,15 @@ webhooks:
     name: pods.credentials-operator.otterize.com
     objectSelector:
       matchLabels:
+        {{- if .Values.global.aws.enabled }}
```

Comment:
Known: this won't work when more than one IAM provider is enabled, but that'll be fixed in a separate task (it requires splitting the webhooks to one for each IAM provider).